packages/glibc/glibc.spec

```diff
@@ -13,19 +13,17 @@ Source3: ldconfig-service.conf
 # applied and reverted during the build.
 Source99: HACK-only-build-and-install-localedef.patch
 
-# Upstream patches from 2.36 release branch:
+# Upstream patches from 2.38 release branch:
```

Comment:
Nit: In the meantime there have been a couple additional patches for malloc. They seem to be optimizations, though, so not critical to pick up right now.

Reply:
I've added these to the stack.